tests/coverage/extensions.py

```diff
@@ -145,43 +145,45 @@ def add_functions_to_map(func_list, function_map, suffix, extension):
     @staticmethod
     def read_substrait_extensions(dir_path: str):
         # read files from extensions directory
-        extensions = []
+        extensions = {}
         for root, dirs, files in os.walk(dir_path):
             for file in files:
                 if file.endswith(".yaml") and file.startswith("functions_"):
-                    extensions.append(os.path.join(root, file))
+                    extensions[os.path.join(root, file)] = '/extensions/' + file
 
-        extensions.sort()
+        extensions = dict(sorted(extensions.items()))
 
         scalar_functions = {}
         aggregate_functions = {}
         window_functions = {}
         dependencies = {}
         # convert yaml file to a python dictionary
-        for extension in extensions:
+        for extension, uri in extensions.items():
```

Comment:
why dict?

Reply:
thought it would be easier to understand. removed as it is unnecessary.